core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.service.ts

```diff
@@ -77,6 +77,14 @@ export class TopicsAndSkillsDashboardPageService {
         if (filterObject.classroom === 'Unassigned' && !topic.classroom) {
           return true;
         }
+
+        if (
+          filterObject.classroom ===
+          TopicsAndSkillsDashboardPageConstants.TOPIC_FILTER_ONLY_CLASSROOMS
```

Comment:
Is this filter different from the TOPIC_FILTER_CLASSROOM_ALL? Based on the implementation - it's not

Reply:
yes it is. TOPIC_FILTER_CLASSROOM_ALL means return all topics.

and TOPIC_FILTER_ONLY_CLASSROOMS indicates just return topics which are assigned to some classroom.